packages/suite/src/components/wallet/getAmountComponent.tsx

```diff
@@ -0,0 +1,46 @@
+import { isNftTokenTransfer, formatAmount, getTxOperation } from '@suite-common/wallet-utils';
+import { TokenTransfer } from '@trezor/connect';
+
+import { FormattedNftAmount } from 'src/components/suite/FormattedNftAmount';
+import { FormattedCryptoAmount } from 'src/components/suite/FormattedCryptoAmount';
+
+type GetAmountComponentProps = {
+    transfer: TokenTransfer;
+    useStyled?: boolean;
+    withLink?: boolean;
+    withSign?: boolean;
+    contract?: string;
+    alignMultitoken?: 'flex-end' | 'flex-start';
+};
+
+export const getAmountComponent = ({
```

Comment:
we discussed yesterday that it should not be `get...`

Reply:
Fixed